### PR TITLE
Feature video convert and audio convert

### PR DESCRIPTION
**Pull request by ramalaso**
Please review for merging.

### CODE REVIEW

File: src/main/java/com/jalasoft/webservice/controller/AudioController.java [hunk 1]
```diff
@@ -1,3 +1,12 @@
...
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
 package com.jalasoft.webservice.controller;
 
 import com.jalasoft.webservice.model.*;
```
Comment (AndyBazualdo) [inline src/main/java/com/jalasoft/webservice/controller/AudioController.java]: import should not have *, use specific imports

File: src/main/java/com/jalasoft/webservice/controller/AudioController.java [hunk 2]
```diff
@@ -7,13 +16,26 @@
 import java.io.IOException;
 
 /**
+ *The class is an endpoint for audio converter
  *
- * @project WebService feature(VideoConvert)
- * @author Isaac Vasquez on 09/23/2019
```
Comment (AndyBazualdo) [inline src/main/java/com/jalasoft/webservice/controller/AudioController.java]: correct autor info

File: src/main/java/com/jalasoft/webservice/controller/AudioController.java (hunk 1 above)
Comment (ramalaso) [reply src/main/java/com/jalasoft/webservice/controller/AudioController.java]: Done

File: src/main/java/com/jalasoft/webservice/controller/AudioController.java (hunk 2 above)
Comment (ramalaso) [reply src/main/java/com/jalasoft/webservice/controller/AudioController.java]: Done

**Review by AndyBazualdo — APPROVED** [review]

**Pull request merged**